internal/pkg/node/methods.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/hpcng/warewulf/internal/pkg/wwlog"
 )
 
+func GetUnsetVerbs() []string {
+	return []string{"UNSET", "DELETE", "UNDEF", "undef", "--", "nil", "0.0.0.0"}
```

Comment:
What would be the correct value to use to unset a mac address?

Reply:
UNDEF works also for mac addresses